testing/test_pypy_bridge_scope.py

```diff
@@ -131,6 +131,79 @@ class C: x = 2
         ''')
         assert php_space.int_w(output[0]) == 2
 
+    def test_php_sees_outer_py_class002(self, php_space):
+        output = self.run('''
+            class C {
+                function __construct() {
+                    $this->x = 3; // should not pick up this class
+                }
```

Comment:
I don't understand why the comment is on this line. Isn't the point that this whole class isn't picked up?